packages/scrapers-v2/src/urls/types.ts

```diff
@@ -0,0 +1,21 @@
+export enum College {
+  ARTS_MEDIA_DESIGN = "arts-media-design",
+  BUSINESS = "business",
+  KHOURY = "computer-information-science",
+  ENGINEERING = "engineering",
+  HEALTH_SCIENCES = "health-sciences",
+  SCIENCE = "science",
+  SOCIAL_SCIENCES_HUMANITIES = "social-sciences-humanities",
+}
+
+export type CatalogHierarchy = {
+  [part: string]: string | CatalogHierarchy;
+};
+
+export type CatalogPath = {
+  // base: https://catalog.northeastern.edu/
+  path: Array<string>;
+  // ex: "undergraduate", "computer-information-science"
+  // or, in the archive case:
+  // ex: "archive", "2018-2019", "undergraduate", "computer-information-science"
```

Comment:
could we move these comments outside and over CatalogPath in a JS multiline comment?
```
/*
Base of the path array: https://catalog.northeastern.edu/
Ex: "undergraduate", "computer-information-science"
or in the archive case,
Ex: "archive", "2018-2019", "undergraduate", "computer-information-science"
*/
```